examples/agentic/dummy_adaptive.py

```
@@ -0,0 +1,139 @@
+class DummyProteinPipeline(ImpressBasePipeline):
+    def __init__(self, name: str, flow: Any, configs: Dict[str, Any] = {}, **kwargs):
+        self.iter_seqs: str = 'MKFLVLACGT'
+        self.generation: int = configs.get('generation', 1)
+        self.parent_name: str = configs.get('parent_name', 'root')
+        self.max_generations: int = configs.get('max_generations', 3)
+        super().__init__(name, flow, **configs, **kwargs)
+
+    def register_pipeline_tasks(self) -> None:
+        @self.auto_register_task()
+        async def sequence_analysis(*args, **kwargs) -> str:
+            return f"/bin/echo 'Analyzing' && /bin/date"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)